pkg/controller/vsphere/machine_scope_test.go

```diff
@@ -320,9 +320,9 @@ func TestPatchMachine(t *testing.T) {
 		},
 	}
 	g.Expect(k8sClient.Create(ctx, configNamespace)).To(Succeed())
-	defer func() {
-		g.Expect(k8sClient.Delete(ctx, configNamespace)).To(Succeed())
-	}()
+	// defer func() {
+	// 	g.Expect(k8sClient.Delete(ctx, configNamespace)).To(Succeed())
+	// }()
```

Comment:
Huh?

Reply:
Yea, this fixed the units locally/ unblocked local development on the pausing stuff. Haven't got around to working out what is actually going on! Will take a look

Reply:
I was hitting the same thing.  it seems we prevent openshift-config from being deleted.